core/src/components/rating/rating.ts

```diff
@@ -208,9 +223,13 @@ export function createRating(config?: PropsConfig<RatingProps>): RatingWidget {
 			// clean inputs with value validation:
 			ariaValueTextFn$,
 
+			ariaLabel$,
+			ariaLabelledBy$,
+
```

Comment:
Instead of adding those stores both here and when calling `stateStores()`, you should instead do as for the `maxRating$, disabled$, readonly$, resettable$` stores:
```ts
const {maxRating$, disabled$, readonly$, resettable$, ariaLabel$, ariaLabelledBy$, className$} = stateProps; 
```